src/Css.elm

```diff
@@ -3809,8 +4003,8 @@ geometricPrecision =
     textRendering optimizeLegibility
 
 -}
-optimizeLegibility: Value { provides | optimizeLegibility : Supported }
-optimizeLegibility=
+optimizeLegibility : Value { provides | optimizeLegibility : Supported }
+optimizeLegibility =
```

Comment:
whoops, I didn't notice this. It must have come in with elm-format. I can separate it into a separate PR if necessary but it seems silly for such a small fix.

Reply:
also kinda ironic that it happened on this particular function 😆

Reply:
it's now fixed on the branch if you want to rebase!

Reply:
done 👍